src/realm-settings/RealmSettingsSection.tsx

```diff
@@ -52,6 +52,19 @@ export const RealmSettingsSection = () => {
     useState<ComponentRepresentation[]>();
   const [currentUser, setCurrentUser] = useState<UserRepresentation>();
   const { whoAmI } = useWhoAmI();
+  const [key, setKey] = useState(0);
+
+  const refresh = () => {
+    setKey(new Date().getTime());
```

Comment:
Instead of using a date this value could also be incremented.
```suggestion
    setKey(key + 1);
```